### DISCUSSION TITLE
Allow scripts which ask for user input

### DISCUSSION THREAD

**Discussion by max-sixty** [go-task/task#438] [Q&A] [answered]
I'm fairly new to Task, so very possible I'm missing something. Thank you for the project — the concept is excellent and I'm already using it in a few projects.

Note this is not the same an https://github.com/go-task/task/pull/236, it's about a script that Task runs being able to take user input, not Task itself.

I have a script that runs a command which asks for user input as part of it running. When I run it normally, it presents a prompt and asks for a token:

```sh
$ gcloud auth login --update-adc --no-launch-browser

Go to the following link in your browser:

    https://accounts.google.com/o/oauth2/auth?response_type=code&client_id={...}

Enter verification code: 
```

But the last line is absent when this is run from go-task:

```yaml
tasks:
  test: gcloud auth login --update-adc --no-launch-browser
```

```sh
$ task test  

task: gcloud auth login --update-adc --no-launch-browser
[test] Go to the following link in your browser:
[test] 
[test]     https://accounts.google.com/o/oauth2/auth?response_type=code&client_id={...}
[test] 
```

Is this intentional?

Thanks!

**Comment by andreynering** [comment, marked as answer]
Hi @max-sixty,

This that you're using [output](https://taskfile.dev/#/usage?id=output-syntax) = prefixed.

Unfortunately we have a limitation where interactive CLI apps only work with the default mode (interleaved). Changing that should make this command work as expected.

**Reply by max-sixty** [reply]
Great, thank you!
